.env.sample

```
@@ -6,4 +6,6 @@ ETHERSCAN_API_KEY=
 OZ_DEFENDER_API_KEY=
 OZ_DEFENDER_API_SECRET=
 # channel's url including key below: https://support.discord.com/hc/en-us/articles/228383668-Intro-to-Webhooks
-DISCORD_URL_WITH_KEY=
\ No newline at end of file
+DISCORD_URL_WITH_KEY=
+DISCORD_URL_WITH_KEY=
+NODE_PROVIDER=
```

```suggestion
  RPC_URL=
```
I think this is what its usually called, so it will be easyer for people to know what it is.